### PR TITLE
Tablet Purchase: fix markdown issues and update

### PR DESCRIPTION
**Pull request by MegaApplePi**
**Description:** part of #962 
**PR Status:** Pending Review

- Guides/TP: fix markdown issues and update
  - include osu!store
    - note that the item is no longer for sale :(
  - include Huion
    - try to write about them
    - include their marketed tablet for osu! players
  - fix somethings in Wacom
  - add images from the original forum post
  - refactor some paragraphs in recommendations section so it doesn't just talk about Wacom
  - adjusted some sentences to go around first person
  - rewrote the forum post link and author
- redirect: add "buying_a_tablet" redirect
  - tablet_purchase just doesn't sound right :/

### CODE REVIEW

File: wiki/Guides/Tablet_Purchase/en.md [hunk 1]
```diff
@@ -1,54 +1,74 @@
-Tablet purchase
-================
+# Tablet purchase
```
Comment (Ephemeralis) [inline wiki/Guides/Tablet_Purchase/en.md]: all major titles should be title case - purchase -> Purchase

File: wiki/Guides/Tablet_Purchase/en.md [hunk 2]
```diff
@@ -1,54 +1,74 @@
...
-**In short:** Tablets are an optional device to let you play the game in a different way. They were originally intended for digital art, but happen to work very well with osu!, and most specifications will make no difference in osu!.
 
-Types of Tablets
-----------------
+The biggest selling point of tablets for osu! is this absolute tracking. Every other parameter such as pressure levels, tilt sensitivity, barrel rotation, etc. will have no effect on your osu! gameplay. These features are for digital art, where the tablet can sense things such as the pressure you're applying to make a darker pencil mark, for example. The only specification which may affect your osu! gameplay is LPI (lines per inch). This is essentially the "resolution" of the tablet surface, or how little you have to move for the tablet to detect it. If a tablet has 100 LPI, then you have to move at least 1/100th of an inch for it to register movement. However, even the lowest LPI should be far higher than any monitor that you're likely to play on, so higher numbers have a mostly unnoticeable effect on gameplay.
```
Comment (Ephemeralis) [inline wiki/Guides/Tablet_Purchase/en.md]: new paragraph at "this is essentially the resolution"

File: wiki/Guides/Tablet_Purchase/en.md [hunk 3]
```diff
@@ -1,54 +1,74 @@
...
+
+tl;dr Tablets are an optional device to let you play the game in a different way. They were originally intended for digital art, but happen to work very well when playing osu!. Most specifications will make no difference in osu!.
```
Comment (Ephemeralis) [inline wiki/Guides/Tablet_Purchase/en.md]: sub tl;dr for "In short,"

File: wiki/Guides/Tablet_Purchase/en.md [hunk 4]
```diff
@@ -1,54 +1,74 @@
...
+
+## Tablet Vendors
+
+### osu!store
+
+*Notice: **The osu!store no longer sells osu!tablets.*** At this time, [peppy has twitted](https://twitter.com/ppy/status/846190638873767936) that if you want to try this input method, you may want to look into other alternatives.
```
Comment (Ephemeralis) [inline wiki/Guides/Tablet_Purchase/en.md]: twitted -> tweeted

File: wiki/Guides/Tablet_Purchase/en.md [hunk 5]
```diff
@@ -1,54 +1,74 @@
...
+
+With the goal of trying to allow players to try the tablet input method without putting a hole in their wallet, the osu!store had featured the osu!tablet. It was made by Huion and designed by flyte. The first version in 2013 featured the osu!tablet (white), a pen with a nib, 3 replaceable nibs, but required a AA battery for the pen (which was included but would add weight to the pen). The second version in 2016 feature the osu!tablet (black), a pen with a nib, 3 replaceable nibs, but the pen needed to be charged via USB (included).
```
Comment (Ephemeralis) [inline wiki/Guides/Tablet_Purchase/en.md]: had featured -> once featured

File: wiki/Guides/Tablet_Purchase/en.md [hunk 6]
```diff
@@ -1,54 +1,74 @@
...
+
+![](img/osu!tabletv1.jpg "osu!tablet v1 \(2013\)")
+
+![](img/osu!tabletv2.jpg "osu!tablet v2 \(2016\)")
+
+### Huion
+
+Huion is a tablet company that is the supplier of the osu!tablet. They are best own to be cheaper than Wacom tablets by a large margin but some would say that their tablets feel flimsier and that running a tablet pen over a Huion tablet would feel as if you were on writing paper (in comparison, some say that on Wacom, it is as if you were writing on glass). Their tablet pens either require a battery or needs to be charged, unlike Wacom tablet pens, and their nibs may need to be often than Wacom nibs. Other than that, there really are not any noticeable differences between Walcom and Huion tablets.
```
Comment (Ephemeralis) [inline wiki/Guides/Tablet_Purchase/en.md]: i have no idea what 'best own to be cheaper' is meant to actually be

this whole paragraph also doesn't really say a lot outside of circumstantial hearsay

File: wiki/Guides/Tablet_Purchase/en.md [hunk 7]
```diff
@@ -1,54 +1,74 @@
...
+At the time of writing there are three versions of the Bamboo, all confusingly under the same "Bamboo" moniker (The Intuos lines are distinguished by number, Intuos5 is the latest at the time of writing). Some would recommend either buying the second or third generation Bamboos, the first generation Bamboo tablets had the extra buttons on the top of the tablet rather than the side, which makes them awkward for use in osu!.
+
+![](img/gen1.jpg "First generation Bamboo (Pen and Touch, Touch, and Fun - Bamboo Fun shown)")
+
+![](http://puu.sh/dlQH "Second generation Bamboo (Pen and Touch, Pen, Touch, Fun, and Craft - Bamboo Pen and Touch shown)")
+
+![](http://puu.sh/dlQi "Third generation Bamboo (Connect, Capture, Create - Connect shown)")
+
+You'll want to stay away from either of the Bamboo Touch tablets, they do not feature pen input, they're just multitouch trackpads for your computer. Frankly, [Apple did it better](http://www.apple.com/magictrackpad/).
+
```
Comment (Ephemeralis) [inline wiki/Guides/Tablet_Purchase/en.md]: new paragraph at 'for osu!'

File: wiki/Guides/Tablet_Purchase/en.md [hunk 8]
```diff
@@ -1,54 +1,74 @@
...
 
-You'll want to stay away from either of the Bamboo Touch tablets - they do not feature pen input, they're just multitouch trackpads for your computer. Frankly, [Apple did it better](http://www.apple.com/magictrackpad/).
-The other Bamboo models come in two sizes - basically small and large. For the second generation, Bamboo Touch, Pen, and Pen and Touch were the "small" ones, whereas the Fun and Craft were the "large" ones. For the third generation, the Connect and Capture are the "small", whereas the Create is the "large". For osu!, you'll probably be looking at getting one of the "small" sized tablets because it's a common strategy to reduce play area so that you don't have to move your arm when you play. However, it's personal preference as to whether or not you want a larger tablet area, and it's especially worth considering the "large" size if you are also into digital art; drawing is much more comfortable on a larger tablet.
+tl;dr Wacom's the recommend brand. Cintiq and Intuos tablets are too expensive and give no real advantage over Bamboo for osu!. See the recommend options above for the recommend models. Also try your luck at a used Intuos3 if you're also into art.
```
Comment (Ephemeralis) [inline wiki/Guides/Tablet_Purchase/en.md]: tl;dr -> "In short, "

probably rephrase to "Wacom is the recommended brand from most serious osu! players"

File: wiki/Guides/Tablet_Purchase/en.md (hunk 6 above)
Comment (MegaApplePi) [reply wiki/Guides/Tablet_Purchase/en.md]: whoops, I thought that was a typo, it was supposed to read:
`They are best known for being cheaper`

I wasn't sure about what else to write about Huion tablets, I was sorta basing this off of another forum and a youtube video

**Pull request merged**